CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# The "code owners" files ensures that each part of the spec is reviewed by subject experts.
+
+# See GitHub docs:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#about-code-owners
+
+# Solana Labs VM team
+/runtime/sbf/ @Lichtso @alessandrod
```

Comment:
@lichtso @alessandrod let me know if you're okay with being added here.

This also requires you to have `write` access to this repo, but that should be easy to do.